policybot/TEST_FLAKES_BOT.md

```diff
@@ -0,0 +1,31 @@
+# Test Flakes Bot design
```

Comment:
Since this actually is the policy bot, it's confusing to call this the test flakes bot. It's really the test flake plugin to the policy bot.

How about just calling this README.md and putting it in policybot/plugins/flakechaser?

Reply:
Make sense. I accidentally include this markdown, originally was just to sync our ideas.

I'll enhance README.md by adding another section, WDTY?

Reply:
How about putting a README file in the individual plugin directory?

Reply:
That seems to be a good idea. Will do.